app/controllers/entries_controller.rb

```diff
@@ -15,6 +15,10 @@ def chat_test
     )
   end
 
```

Comment:
hello please add if statement to create method for validation @GorillaDunk - thank you!!